sortfilterproxymodel.qbs

```diff
@@ -46,6 +47,7 @@ Product
             return defines
         }
 
+        cpp.linkerFlags: "--no-as-needed"
```

Comment:
@BaCaRoZzo @saidinesh5 you guys surely have more experience than me or @crozeg when it comes to new, external QML modules or plugins. How do we do this "the right way"? Do you guys have any ideas? I really don't like this `--no-as-needed` hack and would like to see a better solution.